covalent/_results_manager/result.py

```diff
@@ -467,17 +473,54 @@ def _write_dispatch_to_python_file(self, directory: str = None) -> None:
                 ).get_deserialized()
                 if function is not None and function.__name__ not in functions_added:
 
-                    dispatch_function += self.lattice.transport_graph.get_node_value(
+                    function_str = self.lattice.transport_graph.get_node_value(
                         nodes, value_key="function_string"
                     )
+                    function_str = self._filter_cova_decorators(function_str)
+
+                    dispatch_function += function_str
                     functions_added.append(function.__name__)
 
         lattice_function_str = convert_to_lattice_function_call(
             self.lattice.workflow_function_string,
             self.lattice.workflow_function.__name__,
             self.inputs,
         )
+        lattice_function_str = self._filter_cova_decorators(lattice_function_str)
         dispatch_function += lattice_function_str
 
         with open(os.path.join(result_folder_path, "dispatch_source.py"), "w") as f:
             f.write(dispatch_function)
+
+    def _filter_cova_decorators(self, function_string: str) -> str:
+        """
+        Given a string representing a function, comment out any Covalent-related decorators.
+
+        Args
+            function_string: A string representation of a workflow function.
+
+        Returns:
+            The function string with Covalent-related decorators commented out.
+        """
+
+        has_cova_decorator = False
+        in_decorator = 0
+        function_lines = function_string.split("\n")
+        for i in range(len(function_lines)):
+            line = function_lines[i].strip()
+            if in_decorator > 0:
+                function_lines[i] = f"# {function_lines[i]}"
+                in_decorator += line.count("(")
+                in_decorator -= line.count(")")
```

Comment:
Codecov says this line is not covered by the tests. Can you check that as well?

Reply:
I've added a test for the entire function.